card/lib/cardio/commands.rb

```diff
@@ -20,7 +20,7 @@ def gem
     def map
       @map ||= {
         new: { desc: "create a new deck", group: :shark, via: :call },
-        seed: { desc: "populate a database", group: :shark, via: :rake },
+        setup: { desc: "populate a database", group: :shark, via: :rake },
         update: { desc: "run data updates", group: :shark, alias: :u, via: :rake },
```

Comment:
Worth noting and discussing if there is concern. The new seeding approach means we can go back to supporting a lot more standard rails rake tasks:

- db:seed 
- db:seed:replant (truncate tables and seed)
- db:setup (create and seed)
- db:reset (drop and setup)

What we were calling "seed" is a lot more like what rails calls "setup", so I thought maybe we should embrace that pattern.  Will need to update more docs if we go forward with this.